tests/manual/accessibility.slint

```diff
@@ -0,0 +1,29 @@
+App := Rectangle {
+  width: 400px;
+  height: 400px;
+
+  layout := VerticalLayout {
+    Rectangle {
+      height: 200px;
+      background: red;
+
+      accessible-role: text;
+      accessible-label: "Upper part";
+      accessible-description: "This is the upper part of the test";
+      accessible-has-focus <=> fs.has-focus;
+
+      fs := FocusScope {}
+    }
+    Rectangle {
+      height: 200px;
+      background: green;
+
+      accessible-role: text;
+      accessible-label: "Lower part";
+      accessible-description: "This is the lower part of the test";
+      accessible-has-focus <=> fs.has-focus
```

Comment:
Did you not remove that property?

Reply:
I found two occurrances in a test still. I'll remove those in an extra patch.